packages/ndla-ui/src/ResourceBox/ResourceBox.tsx

```diff
@@ -0,0 +1,205 @@
+  border-radius: 32px;
+  font-weight: 600;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin-bottom: -10px;
+  position: aboslute;
+  border: 1px solid #184673;
+  :hover {
+    background-color: #20588f;
+    color: white;
+  }
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    width: 210px;
+  }
+`;
+const NewLaunchIcon = styled(Launch)`
+  margin-left: 8px;
+  height: 15px;
+  width: 15px;
+  ${mq.range({ until: breakpoints.mobileWide })} {
+  }
+`;
+
+const BoxImage = styled(Image)`
+  object-fit: cover;
+  width: 130px;
+  border-radius: 5px;
+  height: 130px;
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    width: 250px;
+    height: 250px;
+    margin-top: 20%;
+  }
+`;
+
+const CaptionSectionWrapper = styled.div`
+  height: 80%;
+  float: left;
+  width: 70%;
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    height: 60%;
+    width: 100%;
+    display: block;
+  }
+`;
+
+const ImageSectionWrapper = styled.div`
+  height: 100%;
+  width: 25%;
+  margin: 0 auto;
+  float: left;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    height: 50%;
+    width: 100%;
+  }
+`;
+
+const LincenseWrapper = styled.div`
+  position: absolute;
+  right: 0;
+  top: 30px;
+  svg {
+    fill: 'blue';
+  }
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    top: 1px;
+  }
+`;
+
+type Props = {
+  image: string;
+  title: string;
+  caption: string;
+  licenseRights: string[];
+  authors?: { name: string }[];
+  locale?: string;
+  url: string;
+
+  hasLinkedVideo?: boolean;
+};
+export const ResourceBox = ({ image, title, caption, licenseRights, locale, authors, hasLinkedVideo, url }: Props) => {
+  const { t } = useTranslation();
+  return (
+    <BoxWrapper>
+      <ImageSectionWrapper>
+        <BoxImage alt={title} src={image} sizes="25, 25" />
+      </ImageSectionWrapper>
+      <CaptionSectionWrapper>
+        <Boxtitle>{title}</Boxtitle>
+        <Boxcaption>{caption}</Boxcaption>
+      </CaptionSectionWrapper>
+      <StyledButtonDiv>
+        <NewStyledButton href={url} target="_blank">
+          {t('license.other.itemImage.ariaLabel')}
+          <NewLaunchIcon aria-hidden />
+        </NewStyledButton>
+      </StyledButtonDiv>
+      <LincenseWrapper>
+        <LicenseByline licenseRights={licenseRights} locale={locale} marginRight fill="#184673">
+          <div {...classes('byline-author-buttons')}>
+            <span {...classes('byline-authors')}>{authors?.map((author) => author.name).join(' ')}</span>
+            <div>
+              {hasLinkedVideo && (
+                <Button borderShape="rounded" outline size="small" type="button" {...classes('toggleAlternativeVideo')}>
+                  <span className="original">{t('figure.button.alternative')}</span>
+                  <span className="alternative hidden">{t('figure.button.original')}</span>
+                </Button>
+              )}
+            </div>
```

Comment:
Denne treng du ikkje her. Dette er noko som brukes for å bytte mellom videoer som er linka i brightcove, og ikkje relevant.
```suggestion
```